src/code/z_message.c

```diff
@@ -2796,10 +2834,9 @@ void Message_SetView(View* view) {
         Message_DrawTextWide(play, gfx);                         \
     } else {                                                     \
         Message_DrawText(play, gfx);                             \
-    }                                                            \
-    (void)0
+    }
 #else
-#define DRAW_TEXT(play, gfx, isCredits) Message_DrawText(play, gfx)
+#define DRAW_TEXT(play, gfx, isCredits) Message_DrawText(play, gfx);
```

Comment:
I'd prefer the ternary here if it works, I think function-style macros which don't accept semicolons should be avoided

Reply:
context https://discord.com/channels/688807550715560050/688851337085190255/1277681153410732203